src/osd.c

```diff
@@ -24,6 +24,29 @@ set_source(cairo_t *cairo, float *c)
 	cairo_set_source_rgba(cairo, c[0], c[1], c[2], c[3]);
 }
 
+/* Draws a border with a specified border width */
+static void
+draw_border(cairo_t *cairo, double width, double height, double border_width)
```

Comment:
Could do `s/border_width/line_width`... just a matter of taste.

Reply:
As `border_width` could theoretically be confused with `width of the whole border`: :heavy_check_mark: